.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,13 @@
+## Contributor guidelines
+
+1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
+1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
+  - [ ] Ensure any relevant doc is up to date (LINKS??)
+1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below. (_this would also contain updated detail later about this new template_)
+
+```release-note
+* Use the release-note-* labels to set the release note state 
+* Clear this block to use the PR title as the release note 
+or 
```

Comment:
More like ineffective attempt to OR the last two options.  See if it's more obvious looking with -OR-?  Fighting the concise but clear battle.